cms/templates/settings.html

```diff
@@ -410,6 +410,24 @@ <h3 id="heading-entrance-exam">${_("Entrance Exam")}</h3>
             </section>
           % endif
 
+          % if settings.FEATURES.get("ENABLE_SELF_PACED_COURSES", False):
+
+            <hr class="divide" />
+
+            <section class="group-settings pacing">
+              <header>
+                <h2 class="title-2">${_("Course Pacing")}</h2>
+                <span class="tip">${_("Set the pacing for this course")}</span>
+              </header>
+
+              <input type="radio" name="self-paced" id="course-pace-self-paced" value="true"/>
+              <label for="course-pace-self-paced">Self-Paced</label>
+              <input type="radio" name="self-paced" id="course-pace-instructor-led" value="false"/>
+              <label for="course-pace-instructor-led">Instructor Led</label>
+            </section>
+
+          % endif
+
```

Comment:
I don't know if this needs to be part of this PR or a follow on story, but are we planning on styling this like the mockups in this story https://openedx.atlassian.net/browse/SOL-1234 ?